### PR TITLE
117 map 18 mobile view

### LINKED ISSUE #117: Map/18 Mobile view
Depends on #116 

As a user I navigate mostly on the go. So the moblie view of the map is key for my expierience. The part of the screen which is to the right (see location overview, navigation view), should be below the map so i can scroll down to it. The information is not critical here, the buttons are essential (and if already implemented, the distance to the location). 

pasted image 0-3

Depends on #116

### PR DESCRIPTION
**Pull request by 23mafi**
closes #117

### CODE REVIEW

File: app/controllers/map_controller.rb [hunk 1]
```diff
@@ -42,4 +42,34 @@ def directions
     res = Net::HTTP.get_response(uri)
     render json: res.body
   end
+
+  def url
+    profile = params[:profile]
```
Comment (NikkelM) [inline app/controllers/map_controller.rb:47]: The code from here to the end of the file is duplicated.

File: app/javascript/OutdoorMap/router.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Router = L.Routing.OSRMv1.extend({
```
Comment (NikkelM) [inline app/javascript/OutdoorMap/router.js:1]: This file is also duplicated and should be deleted

File: app/javascript/packs/outdoor_map_builder.js [hunk 3]
```diff
@@ -172,6 +172,7 @@ window.routingControl = L.Routing.control({
 // when routing call happens, there will be the stop button in the navigation plan
 .on('routingstart', (e)=>{
     document.getElementById('StopNavigation').style.display = 'block';
+    document.getElementById('welcome-routing-text').style.display = 'none';
```
Comment (NikkelM) [inline app/javascript/packs/outdoor_map_builder.js:175]: In the beginning I was a little confused as to what this does, maybe you could update the name to refer to this being for the mobile view?

**Review by NikkelM — CHANGES_REQUESTED** [review]
Some duplicated code needs to be removed.

File: app/javascript/OutdoorMap/router.js (hunk 2 above)
Comment (felix-20) [reply app/javascript/OutdoorMap/router.js]: fixed it in 4860ac08b0adf6acb7cde1ddef0a22d1b10063d5

File: app/controllers/map_controller.rb (hunk 1 above)
Comment (felix-20) [reply app/controllers/map_controller.rb]: fixed it in 4860ac08b0adf6acb7cde1ddef0a22d1b10063d5

**Review by NikkelM — APPROVED** [review]
Looks good to me, just take a look at the Codefactor I guess

**Pull request merged**